components/checkbox/src/auro-checkbox.js

```diff
@@ -243,13 +250,13 @@ export class AuroCheckbox extends LitElement {
           @input="${this.handleInput}"
           ?disabled="${this.disabled}"
           .checked="${this.checked}"
-          id="${ifDefined(this.id)}"
+          id="${this.inputId}"
```

Comment:
Okay, just so I understand this correctly, the manual `id` was removed from the API, and a custom-generated ID is always generated for the component.